packages/api-derive/src/society/members.ts

```diff
@@ -36,11 +36,16 @@ function _membersPrev (api: DeriveApi, accountIds: AccountId[]): Observable<Deri
 }
 
 function _membersCurr (api: DeriveApi, accountIds: AccountId[]): Observable<DeriveSocietyMember[]> {
+  let currentChallengeRound: u32 = api.registry.createType('u32', 0);
+  api.query.society.challengeRoundCount().subscribe(result => {
+    currentChallengeRound = result;
+  });
+
   return combineLatest([
     of(accountIds),
     api.query.society.members.multi(accountIds),
     api.query.society.payouts.multi(accountIds),
-    api.query.society.defenderVotes.multi(accountIds),
+    api.query.society.defenderVotes.multi(accountIds.map(accountId => [currentChallengeRound, accountId])),
```

Comment:
```suggestion
    api.query.society.challengeRoundCount().pipe(
      switchMap((round) =>
        api.query.society.defenderVotes.multi(accountIds.map(accountId => [round, accountId]))
      )
    ),
```

I believe this should address the issue better - the round is retrieved and then fed to the query. (And all return results at the same time). This gets around the issue where we do a query for round 0 first.

Reply:
Nice suggestion! Looks cleaner.